src/hoverifier.ts

```diff
@@ -44,11 +45,14 @@ import {
     getTokenAtPosition,
     HoveredToken,
 } from './token_position'
-import { HoverAttachment, HoverOverlayProps, isPosition, LineOrPositionOrRange } from './types'
+import { HoverAttachment, HoverOverlayProps, isPosition, LineOrPositionOrRange, DocumentHighlight } from './types'
 import { emitLoading, MaybeLoadingResult, LOADING } from './loading'
 
 export { HoveredToken }
 
+const selectionHighlightClass = 'selection-highlight'
+const documentHighlightClass = 'sourcegraph-document-highlight'
```

Comment:
This would be good to add as options to `HoverifierOptions`.

Would also change the suffix `class`->`className` because that's the conventional name (since `class` is a keyword).